config/test/test-setup.ts

```diff
@@ -35,7 +35,6 @@ if (typeof window !== "undefined") {
     // Override the window.location implementation to mock out assign()
     // We need to access window.location.assign to verify that we're
     // redirecting to the right place.
-    /* eslint-disable no-restricted-syntax */
```

Comment:
Some file-wide exceptions seemed to be for nothing, but our linter doesn't seem to find useless file-wide exceptions.